packages/client-core/src/user/services/AuthService.ts

```diff
@@ -329,7 +321,7 @@ export const AuthService = {
       }
     }
   },
-  loginUserByOAuth: async (service: string) => {
+  loginUserByOAuth: async (service: string, location: any) => {
```

Comment:
should we just use useLocation inside loginUserByOAuth instead of passing it in?

Reply:
i tried using the useLocation hook in service file initially but it was crashing and throwing error in console.